yb-voyager/cmd/config.go

```diff
@@ -213,6 +222,16 @@ func initConfig(cmd *cobra.Command) ([]ConfigFlagOverride, []EnvVarSetViaConfig,
 	// CLI Flag > ENV Variable > Default config file in home directory
 	if cfgFile != "" {
 		// Use config file from the flag.
+		if !utils.FileOrFolderExists(cfgFile) {
+			return nil, nil, nil, fmt.Errorf("config file does not exist: %s", cfgFile)
+		}
+
+		cfgFile, err := filepath.Abs(cfgFile)
+		if err != nil {
+			return nil, nil, nil, fmt.Errorf("failed to get absolute path for config file: %s: %w", cfgFile, err)
+		}
+		cfgFile = filepath.Clean(cfgFile)
```

Comment:
what does this do, and why do we need it?

Reply:
It basically sanitizes and normalizes the cfgFile path string. It removes any redundant separators (like multiple slashes /) and relative components like . and .. . Even after you get the absolute path using filepath.Abs(cfgFile), the path string might still contain redundant or relative components. 
This is something that we do to the exportDir and other paths as well. You can check `validateExportDirFlag()` and `validateMoveToFlag()` etc